setup.py

```diff
@@ -21,5 +21,6 @@
       package_data={
           'fluent.migrate': ['cldr_data/*']
       },
-      tests_require=['six']
+      tests_require=['six'],
```

Comment:
I tried adding `compare-locales` here but `python setup.py test` fails with version conflicts when it tries to install it. `compare-locales` requires `fluent==0.4.4` while `python-fluent`'s master is usually newer than that.